### PR TITLE
Add simple notebook smoke test

### PR DESCRIPTION
**Pull request by lucyzhang929**
This smoke test creates a new notebook, configures Python, and executes cell.

### CODE REVIEW

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]
Would rather we not use aria labels to select the elements.

Did you run through testing what happens when things fail? (can't find an element, can't change to python kernel, python installation fails, etc)

File: test/automation/src/sql/configurePythonDialog.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	constructor(code: Code) {
+		super(CONFIGURE_PYTHON_DIALOG_TITLE, code);
+	}
+
+	async waitForConfigurePythonDialog(): Promise<void> {
+		await this.waitForNewDialog();
+	}
+
+	async installPython(): Promise<void> {
+		await this.code.waitAndClick('.modal');
```
[the remark this answers is not in the document]
Comment (lucyzhang929) [reply test/automation/src/sql/configurePythonDialog.ts:22]: I can make this more specific - just want to make sure that the dialog is in focus before doing anything else.

File: test/automation/src/sql/configurePythonDialog.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+		const newPythonInstallation = '.modal .modal-body input[aria-label="New Python installation"]';
```
[the remark this answers is not in the document]
Comment (lucyzhang929) [reply test/automation/src/sql/configurePythonDialog.ts:24]: Will look into adding ids for elements

**Comment by lucyzhang929** [comment]
I added ids for the notebook kernel dropdown and attach to dropdown. I think adding ids for the wizard buttons will require more changes - I'll keep looking into it.

As for failures, the smoke test will time out after 20 seconds and fail if it can't find an element or something fails to happen (can't find an element, can't change to python kernel, python installation fails, etc). Right now the test is only checking that there is an output after executing the cell. I will be adding tests that check the actual output is correct in the future.

**Review by Charles-Gagnon — APPROVED** [review]
It would be great to use IDs for everything - but that might take more work so might just be better to create a separate task for and we can follow up with that later.

**Pull request merged**